infrastructure/terraform/components/branch/module_amplify_branch.tf

```diff
@@ -1,19 +1,19 @@
 module "amplify_branch" {
-  source = "../../modules/amp_branch"
-
+  source         = "git::https://github.com/NHSDigital/nhs-notify-shared-modules.git//infrastructure/modules/amp_branch?ref=v1.0.0"
   name           = local.normalised_branch_name
   aws_account_id = var.aws_account_id
   component      = var.component
   environment    = var.environment
   project        = var.project
   region         = var.region
   group          = var.group
-
-  cognito_user_pool_client_id               = local.iam.cognito_user_pool_client["id"]
-  cognito_user_pool_identity_provider_names = local.iam.cognito_user_pool["identity_providers"]
-  amplify_app_id                            = local.iam.amplify["id"]
-  branch                                    = var.branch_name
-  domain_name                               = local.root_domain_name
-  subdomain                                 = var.environment
-  base_path                                 = "/auth~${local.normalised_branch_name}"
+  description    = "Amplify branch for ${local.normalised_branch_name}"
+  amplify_app_id = local.iam.amplify["id"]
+  branch         = var.branch_name
+  display_name   = local.normalised_branch_name
+  environment_variables = {
```

Comment:
youll need to run the terraform fmt again

Reply:
weirdly, running terraform fmt doesn't seem to think it needs to change  this 🤔